source/puppet/5.3/hiera_migrate.md

```diff
@@ -69,70 +69,73 @@ You can enable the environment layer gradually. In migrated environments, the le
 
 The process of enabling the environment layer involves the following steps, each of which is described in the instructions below. In each environment, you will:
 
-1. Check your code for Hiera function calls with "hierarchy override arguments" (as shown later), which will cause errors.
+1.  Check your code for Hiera function calls with "hierarchy override arguments" (as shown later), which will cause errors.
 
-2. Add a local `hiera.yaml` file.
+2.  Add a local `hiera.yaml` file.
 
-3. Update your custom backends if you have them.
+3.  Update your custom backends if you have them.
 
-4. Rename the data directory to exclude this environment from the global layer. Unmigrated environments still rely on the     global layer, which gets checked before the environment layer. If you want to maintain both migrated and unmigrated environments during the migration process, choose a different data directory name for migrated environments. The new name 'data' is a good choice because it is also the new default (unless you are already using 'data', in which case choose a different name and set datadir in hiera.yaml). 
+4.  Rename the data directory to exclude this environment from the global layer. Unmigrated environments still rely on the global layer, which gets checked before the environment layer. If you want to maintain both migrated and unmigrated environments during the migration process, choose a different data directory name for migrated environments. The new name 'data' is a good choice because it is also the new default (unless you are already using 'data', in which case choose a different name and set datadir in hiera.yaml).
 
-   This process has no particular time limit and shouldn’t involve any downtime. Once all of your environments are migrated, you can phase out or greatly reduce the global hierarchy.
+    This process has no particular time limit and shouldn't involve any downtime. Once all of your environments are migrated, you can phase out or greatly reduce the global hierarchy.
```

Comment:
Also applied consistent four-character indents to paragraphs and blockquotes/notes within a list item.